src/index.html

```diff
@@ -15,30 +17,28 @@ <h1>LEADERBOARD</h1>
 
```

Comment:
### Your Summary
![summary-format-fixable](https://user-images.githubusercontent.com/93039359/180229472-9b62e602-273e-4ef6-acf5-75f64308979b.png)

### Proper Format Summary Example :heavy_check_mark:
![summary](https://user-images.githubusercontent.com/93039359/180229523-e9308341-78cf-4a6a-9780-0ab33aa959a3.png)

- [ ] You have a descriptive summary but it would be nice if you add your summary in the format shown above. :+1: Because it shows your professionalism and attract the recruiters :thinking: